changes/955.feature.md

```diff
@@ -1 +1 @@
-Added support for GET /users/@me/guilds/{guild}/member
+Added support for GET `/users/@me/guilds/{guild}/member`.
```

Comment:
Shouldn't get also be in the code blocks also why put it in code blocks if it's not referring to anything in code

Reply:
Its just for formatting, but ig it can be left how it was